apm-agent-core/src/main/java/co/elastic/apm/agent/impl/context/web/WebConfiguration.java

```diff
@@ -58,7 +58,8 @@
 
     private final ConfigurationOption<List<WildcardMatcher>> ignoreUrls = ConfigurationOption
         .builder(new ListValueConverter<>(new WildcardMatcherValueConverter()), List.class)
-        .key("ignore_urls")
+        .key("transaction_ignore_urls")
+        .aliasKeys("ignore_urls")
```

Comment:
Wow, you can even do an alias in your config-system; very fancy! 👏 👏

Reply:
Yeah, we use a fancy third party for that 😉

Reply:
Let's call it a 2.5 party 😛